tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -16,7 +16,7 @@ import (
 	"reflect"
 	"strconv"
 
-	"github.com/antlr/antlr4/runtime/Go/antlr"
+	"github.com/cyberfox/antlr4/runtime/Go/antlr"
```

Comment:
❗️ These lines were used for testing but cannot change like this in the main repository.